src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random number = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private int lowerBound = 1; // minimum value of ball roll 1
+    private int upperBound = 101; // maximum value of ball roll 100

```

Comment:
Read about constants and how to create them in Java - comments are not a good practice